oracle/fuel-toolchain.toml

```
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "beta-1"
```

```suggestion
channel = "latest-2023-01-18"
```

I don't know about that one chief

---

Ughh I was testing locally, thanks for catching this.